app/routes/cache.js

```diff
@@ -0,0 +1,19 @@
+const getSommit = require('../get-sommit')
+const { set } = require('../cache')
+
+module.exports = [{
+  method: 'GET',
+  path: '/get',
+  handler: async (request, h) => {
+    const res = await getSommit(request, 'Web')
+    return h.response(res).code(200)
+  }
+},
+{
+  method: 'GET',
+  path: '/set',
+  handler: async (request, h) => {
+    await set(request, 'Web', `Value gotten from web get with timestamp ${new Date().toString()}`)
+    return h.response('ok').code(200)
+  }
+}]
```

Comment:
Do these routes need to exist?

Reply:
Handing over to Simon and note on this.

TL;DR I suggest keeping these routes in the main branch to help test in DEV and TEST.

This isn't currently in DEV or TEST due to missing ACR instances which I _assume_ will come when it merges into `main` <-- 1st thing to confirm.

Anyway, because of this I'm thinking these should be kept in to allow this (with the routes) to be elevated to DEV where we can use these to check the caching is working as it did in SND (same Redis config and instance so quietly confident). Just means this will give us an easy way to test in these environments.

Then in a follow-up ticket which is to basically merge Abi's (API returning blob statement) and mine (caching) work pieces together we can remove the routes then?

Reply:
Yes that makes sense.